mibs/MITEL-APPCMN-MIB.txt

```diff
@@ -0,0 +1,225 @@
+-- Copyright 2004, 2005, 2006 MITEL Networks Corporation
```

Comment:
please create `mib/mitel` and add this mib to that folder, filename needs to be `MITEL-APPCMN-MIB`